NGitLab.Mock/Repository.cs

```diff
@@ -558,14 +584,27 @@ internal string FetchBranchFromFork(Project fork, string branch)
         return $"remotes/{forkRemote.Name}/{branch}";
     }
 
-    private static Tree GetTreeItem(string repositoryPath, string filePath)
+    private Tree GetTreeItem(string repositoryPath, string filePath)
```

Comment:
I'm working on refactoring this method to more accurately match what GitLab does.

Reply:
done